### PR TITLE
Corrected the set value of the HEIGHT value

### PR DESCRIPTION
**Pull request by dongrie**
* Fixed headers at CreateMsgCreateClient and GetLatestFinalizedHeader
* WithProof to add makeProof

### CODE REVIEW

File: provers/mock/prover.go [hunk 1]
```diff
@@ -84,7 +86,7 @@ func (pr *Prover) QueryClientConsensusStateWithProof(ctx core.QueryContext, dstC
 		return nil, err
 	}
 	res.Proof = makeProof(bz)
-	res.ProofHeight = clienttypes.NewHeight(0, pr.sequence)
+	res.ProofHeight = clienttypes.NewHeight(0, ctx.Height().GetRevisionHeight())
```
[the remark this answers is not in the document]
Comment (siburu) [reply provers/mock/prover.go:89]: Or, is it OK to just use `ctx.Height()` as `res.ProofHeight`, too?

[the remark this answers is not in the document]
Comment (dongrie) [reply provers/mock/prover.go]:
```
res.ProofHeight = ctx.Height().(clienttypes.Height)
```
Fixed it this way
https://github.com/hyperledger-labs/yui-relayer/pull/86/commits/7407616d994f58a93e19fb8c3edd4827a1149b1b

**Comment by bluele** [comment]
@dongrie 
LGTM, so could you squash the commits to a few meaning commits?

**Comment by dongrie** [comment]
@bluele 
Thank you. I squashed multiple commits into one

**Review by bluele — APPROVED** [review]

**Pull request merged**